<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_session_locale</class>
 <widget class="QWidget" name="page_session_locale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_30">
     <property name="text">
      <string>System localization settings (restart required)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="combo_locale_lang"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Messages</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="combo_locale_message"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="combo_locale_time"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Numeric</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="combo_locale_numeric"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Monetary</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="combo_locale_monetary"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Collate</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="combo_locale_collate"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>CType</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="combo_locale_ctype"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
